code/datums/wounds/bones.dm

```diff
@@ -104,120 +102,91 @@
 			limb.receive_damage(brute=rand(3,7))
 			return COMPONENT_NO_ATTACK_HAND
 
-/datum/wound/brute/bone/receive_damage(wounding_type, wounding_dmg, wound_bonus)
+/datum/wound/blunt/receive_damage(wounding_type, wounding_dmg, wound_bonus)
```

Comment:
is it intentional that there's no threshold for how high wounding_dmg has to be to trigger blood loss for blunt chest wounds, even though such a threshold exists for piercing wounds?

Reply:
done